src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  position: fixed;
```

Comment:
if we use BEM here, we are not able to  give position, margins and width to BEM-block